crates/shared/src/encoded_settlement.rs

```diff
@@ -84,6 +85,65 @@ pub struct EncodedSettlement {
     pub interactions: [Vec<EncodedInteraction>; 3],
 }
 
+impl EncodedSettlement {
+    /// Order uids for all trades in the settlement.
+    ///
+    /// Returns order uids for all trades in the settlement in the same order as
+    /// the trades.
+    pub fn uids(
```

Comment:
Not used in this PR. Are you planning on using that in a future PR?

Reply:
Not needed anymore. Removed 👍